packages/arb-bridge-eth/contracts/rollup/ArbRollup.sol

```diff
@@ -283,6 +287,46 @@ contract ArbRollup is IArbRollup, Cloneable, NodeGraph, Staking {
         safeSelfDestruct(msg.sender);
     }
 
+    function ownerToggleStakerAllowListed(bool shouldRequire) external onlyOwner {
```

Comment:
I think this should be called `ownerToggleStakerAllowList`.

Reply:
I'd change this, but I already deployed it